Packs/VMware/Integrations/VMware/VMware.yml

```diff
@@ -0,0 +1,233 @@
+    description: Issues a command to the guest operating system asking it to perform
+      a reboot.
+    name: vmware-soft-reboot
+  - arguments:
+    - description: VM UUID of virtual machine to take snapshot of.
+      name: vm-uuid
+      required: true
+    - description: Snapshot name
+      name: name
+    - description: Snapshot description
+      name: description
+    - auto: PREDEFINED
+      defaultValue: 'True'
+      description: Snapshot the virtual machine's memory
+      name: memory
+      predefined:
+      - 'true'
+      - 'false'
+    - auto: PREDEFINED
+      defaultValue: 'False'
+      description: Quiesce guest file system (needs VMWare Tools installed).
+      name: quiesce
+      predefined:
+      - 'true'
+      - 'false'
+    description: Creates VM snapshot
+    name: vmware-create-snapshot
+  - arguments:
+    - description: Snapshot name to revert to
+      name: snapshot-name
+      required: true
+    - description: VM UUID of virtual machine to revert snapshot.
+      name: vm-uuid
+      required: true
+    description: Reverts VM to snapshot
+    execution: true
+    important:
+    - contextPath: VMWare.UUID
+      description: VM instance UUID
+      related: ''
+    - contextPath: VMWare.Snapshot
+      description: Name of the snapshot reverted to.
+      related: ''
+    name: vmware-revert-snapshot
+    outputs:
+    - contextPath: VMWare.UUID
+      description: VM instance UUID
+    - contextPath: VMWare.Snapshot
+      description: Name of the snapshot reverted to.
+  - arguments:
+    - description: VM UUID of virtual machine to get events of.
+      name: vm-uuid
+      required: true
+    - defaultValue: VmGuestRebootEvent,VmGuestShutdownEvent,VmPoweredOnEvent,VmPoweredOffEvent,VmSuspendedEvent
+      description: Type of events to get, given in CSV (i.e.  VmGuestRebootEvent,VmGuestShutdownEvent).
+      name: event-type
+    description: Gets events of VM
+    name: vmware-get-events
+  - arguments:
+    - description: VM UUID of virtual machine to change NIC state
+      name: vm-uuid
+      required: true
+    - auto: PREDEFINED
+      description: New state of the NIC to be changed
+      name: nic-state
+      predefined:
+      - connect
+      - disconnect
+      - delete
+      required: true
+    - description: Number of the NIC to be changed
+      name: nic-number
+      required: true
+    description: Changes the state of a VM NIC
+    important:
+    - contextPath: VMWare.UUID
+      description: VM instance UUID
+      related: ''
+    - contextPath: VMWare.NICState
+      description: VM NIC State (i.e. connected, disconnected, delete)
+      related: ''
+    name: vmware-change-nic-state
+    outputs:
+    - contextPath: VMWare.UUID
+      description: VM instance UUID
+    - contextPath: VMWare.NICState
+      description: VM NIC State (i.e. connected, disconnected, delete)
+  dockerimage: demisto/vmware:1.0-alpine
+  runonce: false
+  script: ''
+  subtype: python3
```

Comment:
```suggestion
  subtype: python2
```

Reply:
Well the raw-response doesnt provide much information and is the same as the context, just like in almost all the commands of the integration:
![image](https://user-images.githubusercontent.com/34412641/99695907-c98f3a00-2a96-11eb-997c-0703111ec7c3.png)